code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -17,7 +17,7 @@
 	icon_state = "m42a"
 	item_state = "m42a"
 	max_shells = 15 //codex
-	caliber = "10x28mm Caseless" //codex
+	caliber = "10x28mm" //codex
```

Comment:
```suggestion
	caliber = "10x28mm"
```

Reply:
The vars having a comment pointing they are codex-related is not a bad thing IMO.

Reply:
It is misleading in making it seems like they're ONLY there for codex.

Reply:
Ah, fair.

Reply:
Either removing the comment or making it more clear is fine, but right now it's not ideal.